test-case-reporting/app.ts

```diff
@@ -67,6 +67,15 @@ function extractPageInfo(req: express.Request): PageInfo {
   };
 }
 
+// We need the testRun index in mustache to give each collapsable
+// its own id.
+function enumerateTestRun(
```

Comment:
Does this method take a single testRun-index pair as input, or does it take a list of testRuns?

Reply:
It takes a single testRun-index pair as an input, because I find `testRuns.map(enumerateTestRun)` to be clearer than `enumerateTestRuns(testRuns)`. I'm open to changing it.

Reply:
```typescript
const testRuns: Array<TestRun> = [/* stuff */];
function enumerateTestRun(testRun: TestRun, index: number) {/* stuff */}
testRuns.map(enumerateTestRun); // Where is the value of index coming from?
```

Reply:
Not sure how I missed this.

I don't understand the suggestion, are you saying `enumerateTestRun` should be declared inside the arrow function of the app.get? It is used by two different blocks so it needs to be declared outside.

Reply:
Cleared up offline, resolving